doc/SAI_1.7.1 Release notes.md

```diff
@@ -0,0 +1,149 @@
+
+`````
+ * @brief Cipher suite for this Secure Channel.
+ *
+ * @type sai_macsec_cipher_suite_t
+ * @flags MANDATORY_ON_CREATE | CREATE_AND_SET
+`````
+SAI_MACSEC_SC_ATTR_MACSEC_CIPHER_SUITE,
+
+The PR related to this feature is available at [PR#1156](https://github.com/opencomputeproject/SAI/pull/1156) ;[PR#1172](https://github.com/opencomputeproject/SAI/pull/1172)
+
+### Tunnel loopback packet action as resource	
+SAI_TUNNEL_ATTR_LOOPBACK_PACKET_ACTION is marked as resource and can be queried using the sai_object_type_get_availability API.
+`````
+Example
+sai_attribute_t *attr;
+sai_status_t status;
+uint64_t count;
+
+attr[0].id = SAI_TUNNEL_ATTR_TYPE;
+attr[0].value.s32 = SAI_TUNNEL_TYPE_VXLAN;
+
+attr[1].id = SAI_TUNNEL_ATTR_LOOPBACK_PACKET_ACTION;
+attr[1].value.s32 = SAI_PACKET_ACTION_DROP;
+
+status = sai_object_type_get_availability(g_switch_id, SAI_OBJECT_TYPE_TUNNEL, 2, &attr, &count);
+`````
+
+This API will return number of vxlan tunnels which can further with packet action as drop.
+SONiC when invokes this API with out any tunnels configured
+SAI will return 4
+SONiC will configure 1 tunnel with packet action as drop and call the query API.
+SAI will return 3 and so forth
+
+The PR related to this feature is available at [PR#1163](https://github.com/opencomputeproject/SAI/pull/1163)
+
+### PFC pause duration in microseconds 
+Pause duration has been added as below
+
+     * PFC pause duration for RX and TX per PFC priority in micro seconds 
+     * RX pause duration for certain priority is a the duration in micro seconds converted
+     * from quanta in ingress pause frame for that priority (a pause frame received by the
+     * switch).
+     * While TX pause duration for certain priority is the duration in micro seconds converted
+     * from quanta in egress pause frame for that priority (a pause frame sent by the switch).
+
+This is acheieved by adding attribute SAI_PORT_STAT_PFC_7_TX_PAUSE_DURATION for enum sai_port_stat_t in saiport.h 
+
+The PR related to this feature is available at [PR#1143](https://github.com/opencomputeproject/SAI/pull/1143); [PR#1144](https://github.com/opencomputeproject/SAI/pull/1144); [PR#1145](https://github.com/opencomputeproject/SAI/pull/1145)
+
+### Query statistics capability   
+Query statistics capability adds the ability to query statistics values capability per object type - meaning which statistics enum values are implemented per object type
+Currently, these is not exposed, and in order to check capabilities, the user has to retry each value, one by one. This is achieved by adding typedef sai_query_stats_capability for saiobject.h also by adding typedef sai_stat_capability_t in saitypes.h 

```

Comment:
instead of typedef sai_query_stats_capability should say
new function sai_query_stats_capability

Reply:
addressed